Makefile

```diff
@@ -1,7 +1,7 @@
 PREFIX            ?= $(shell pwd)
 FILES             ?= $(shell find . -type f -name '*.go' -not -path "./vendor/*")
 DOCKER_IMAGE_NAME ?= thanos
-DOCKER_IMAGE_TAG  ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))
+DOCKER_IMAGE_TAG  ?= $(subst /,-,$(shell git rev-parse --abbrev-ref HEAD))-$(shell date +%Y-%m-%d)-$(shell git rev-parse HEAD)
```

Comment:
do you want the full hash here or would `git rev-parse --short HEAD` be good enough?

Reply:
`master-2018-05-14-102c8f66af6b9dfde93e9ad07b8bd084b8e4417d` is what the tag would currently look like

Reply:
yea, we can live with short version I guess